src/main/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResource.java

```diff
@@ -51,11 +51,11 @@ public class CollectionResource {
 
   public static final String BASE_PATH = "/v1/{namespace}/{collection}";
 
-  private final CommandProcessor commandProcessor;
+  private final MeteredCommandProcessor meteredCommandProcessor;
 
   @Inject
-  public CollectionResource(CommandProcessor commandProcessor) {
-    this.commandProcessor = commandProcessor;
+  public CollectionResource(MeteredCommandProcessor meteredCommandProcessor) {
```

Comment:
There was no need to change this to metered processor, you could have just removed the `@ApplicationScoped` from the `CommandProcessor`.. Then `MeteredCommandProcessor` would be the only `CommandProcessor` in the app that could be injected..

This could in future allow us to disable processor metering using the config property and have a configuration method that deciedes what instance is used.

Reply:
Discussed with @ivansenic, leaving it as it is.